Wrapping up v3.0.0 changes (#16)
eBay/NSTSuite: TMBuilder/README.md

| 1.0.0 | Support for OpenAPI schema |
</details>

## Thin Models

> Thin models are model classes supported by JSON path expressions for retrieving leaf node values. Instead of a full POJO structure requiring lengthy dot notation to retreive nested properties, JSON path allows for retrieval of a node or set of nodes to perform an verification of data correctness. This is the second order contract testing check performed where the data of the response is evaluated for correctness as defined in the thin model checks. Once you've defined all of your checks you may export the tests to your check class and the TMBuilder will generate the code for each check you've defined in the TMBuilder.

Reviewer: ```suggestion
Thin models are model classes supported by JSON path expressions for retrieving leaf node values. Instead of a full POJO structure requiring lengthy dot notation to retreive nested properties, JSON path allows for retrieval of a node or set of nodes to perform an verification of data correctness. This is the second order contract testing check performed where the data of the response is evaluated for correctness as defined in the thin model checks. Once you've defined all of your checks, you can generate the code for each check you've defined by exporting them to an associated check file.
```

$0.02

Author: Done.
